W-18195573-CHtoCH2-App-Migration (#787)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/ch-ch2-migration-completion.adoc

= CloudHub to CloudHub 2.0 Migration Completion

> After completing the VPC upgrade and application migration phases, take these final steps to complete the CloudHub to CloudHub 2.0 migration process and release associated entitlements within the recommended time frame:

Reviewer: ```suggestion
After completing the VPC upgrade and application migration phases, complete the CloudHub to CloudHub 2.0 migration process and release associated entitlements within the recommended time frame:
```

== Mark Upgrade as Complete

Marking the upgrade as complete finishes the entire VPC upgrade process. Go to the *Upgrade* tab and click *Mark Upgrade Complete*.

image::mark-upgrade-complete.png[*Mark Upgrade Complete* button within the Upgrade tab]

After you mark the upgrade complete, CloudHub 2.0 replaces all CloudHub resources and CloudHub assets become immutable. Perform any required changes on CloudHub 2.0.

> [WARNING] 

Reviewer: Please move this to right below the heading.

> This is an irreversible action. Clicking *Mark Upgrade Complete* deletes the original VPC, stopped DLBs, and all CloudHub 1.0 applications marked as upgraded. All traffic then flows through the CloudHub 2.0 private space.

Reviewer: ```suggestion
Marking the upgrade as complete is an irreversible action. This action deletes the original VPC, stopped DLBs, and all CloudHub 1.0 applications marked as upgraded. All traffic then flows through the CloudHub 2.0 private space.
```

== Entitlements for VPCs, Private Spaces, and Network Connections (VPNs/TGW)

After you complete the VPC upgrade and bring over any existing TGW and VPN network connections, your old and new infrastructure count separately against your usage. Therefore, if your organization uses all purchased quota, your usage can exceed the allowed quota until you move all your applications to CloudHub 2.0 and decommission your old VPC.

Work with your account representative to plan for additional entitlements, and decommission your CloudHub infrastructure (VPCs and associated DLBs) within six months after the upgrade to avoid interruptions when trying to create or edit VPCs, private spaces, or network connections.

[NOTE]

> Deleting the upgraded VPC once all applications and their associated DLBs are removed doesn't affect your CloudHub 2.0 Private Spaces or CloudHub 2.0 applications.

Reviewer: ```suggestion
Deleting the upgraded VPC after all applications and their associated DLBs are removed doesn't affect your CloudHub 2.0 Private Spaces or CloudHub 2.0 applications.
```
